{
 "PRed": [
  {
   "PR": {
    "__lazy_json__": "pr_json/1271911528.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "MigrationYaml",
    "migrator_object_version": 1,
    "migrator_version": 0,
    "name": "python311"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_object_version",
    "migrator_version",
    "name"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/1272514509.json"
   },
   "data": {
    "bot_rerun": 1679924224.1623476,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.0.1.1"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/1292903209.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.0.1.1"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR": {
    "__lazy_json__": "pr_json/1296073006.json"
   },
   "data": {
    "bot_rerun": false,
    "migrator_name": "Version",
    "migrator_version": 0,
    "version": "4.0.1.2"
   },
   "keys": [
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  }
 ],
 "bad": false,
 "pinning_version": "2023.03.29.16.02.14",
 "pre_pr_migrator_attempts": {
  "pypy38": 22
 },
 "pre_pr_migrator_status": {
  "pypy38": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/4647406008\">bot CI job</a>): main: ['linux_64_python3.8.____73_pypyqt5: Encountered problems while solving:\\n  - package pyside2-5.13.2-py38h3b90aee_2 requires qt >=5.12.9,<5.13.0a0, but none of the providers can be installed\\n', 'linux_64_python3.8.____73_pypyqt6: Encountered problems while solving:\\n  - package pyside6-6.4.0-py37hac12498_0 requires python >=3.7,<3.8.0a0, but none of the providers can be installed\\n', 'linux_64_python3.9.____73_pypyqt5: Encountered problems while solving:\\n  - package pyside2-5.13.2-py39h9c294d1_3 requires qt >=5.12.9,<5.13.0a0, but none of the providers can be installed\\n', 'linux_64_python3.9.____73_pypyqt6: Encountered problems while solving:\\n  - package pyside6-6.4.0-py37hac12498_0 requires python >=3.7,<3.8.0a0, but none of the providers can be installed\\n', 'osx_64_python3.8.____73_pypyqt5: Encountered problems while solving:\\n  - package python-3.8.12-0_73_pypy requires python_abi 3.8 *_pypy38_pp73, but none of the providers can be installed\\n', 'osx_64_python3.8.____73_pypyqt6: Encountered problems while solving:\\n  - package pyside6-6.4.0-py37hd4b7e59_0 requires python >=3.7,<3.8.0a0, but none of the providers can be installed\\n', 'osx_64_python3.9.____73_pypyqt5: Encountered problems while solving:\\n  - package pyside2-5.13.2-py39h717a4ca_3 requires qt >=5.12.9,<5.13.0a0, but none of the providers can be installed\\n', 'osx_64_python3.9.____73_pypyqt6: Encountered problems while solving:\\n  - package pyside6-6.4.0-py37hd4b7e59_0 requires python >=3.7,<3.8.0a0, but none of the providers can be installed\\n', 'win_64_python3.8.____73_pypyqt5: Encountered problems while solving:\\n  - package pyside2-2.0.0.dev0-py35_vc14_0 requires vc 14, but none of the providers can be installed\\n', 'win_64_python3.8.____73_pypyqt6: Encountered problems while solving:\\n  - package pyside6-6.4.0-py37h1ccf79b_0 requires python >=3.7,<3.8.0a0, but none of the providers can be installed\\n', 'win_64_python3.9.____73_pypyqt5: Encountered problems while solving:\\n  - package pyside2-2.0.0.dev0-py35_vc14_0 requires vc 14, but none of the providers can be installed\\n', 'win_64_python3.9.____73_pypyqt6: Encountered problems while solving:\\n  - package pyside6-6.4.0-py39hfa61e5a_0 requires libclang >=14.0.6,<15.0a0, but none of the providers can be installed\\n']"
 },
 "smithy_version": "3.23.1"
}